scripts/benchmarks_ci.py

```diff
@@ -407,20 +407,20 @@ def __run_benchview_scripts(
     if 'Configuration' not in benchview_config:
         benchview_config['Configuration'] = args.configuration
 
-    # Generate existing configs.
-    # TODO: Unify configs across all repos?
-    submission_architecture = args.architecture
-    if args.category.casefold() == 'CoreClr'.casefold():
-        benchview_config['JitName'] = 'ryujit'  # FIXME: Remove this.
-        benchview_config['OS'] = __get_coreclr_os_name()
-        benchview_config['OptLevel'] = args.optimization_level  # Optimization
-        benchview_config['PGO'] = 'pgo'  # FIXME: Remove this? Enabled/Disabled
-        benchview_config['Profile'] = 'On' if args.enable_pmc else 'Off'
-    elif args.category.casefold() == 'CoreFx'.casefold():
-        submission_architecture = 'AnyCPU'
-        benchview_config['OS'] = __get_corefx_os_name()
-        benchview_config['RunType'] = \
-            'Diagnostic' if args.enable_pmc else 'Profile'
+    # Generate configurations.
+    def __get_os_name():
+        if sys.platform == 'win32':
+            return '{} {}'.format(platform.system(), platform.release())
+        elif sys.platform == 'linux':
+            os_name, os_version, _ = platform.linux_distribution()
+            return '{}{}'.format(os_name, os_version)
+        else:
+            return platform.platform()
+
+    benchview_config['Jit'] = 'RyuJIT'  # TODO: Hardcoded Jit name.
```

Comment:
I think we should have `JIT` is some way, but at the moment I do not know how this is applicable to `Mono` and `CoreRT`. This is why I have left it there.

Reply:
Mono does have multiple JIT engines.  I'd read this as an optional value, and so you don't need to specify it here, but it can be specified if necessary/when applicable.